homeassistant/components/dublin_bus_transport/sensor.py

```diff
@@ -45,23 +46,11 @@
 )
 
 
-def due_in_minutes(timestamp):
-    """Get the time in minutes from a timestamp.
-
-    The timestamp should be in the format day/month/year hour/minute/second
-    """
-    diff = datetime.strptime(timestamp, "%d/%m/%Y %H:%M:%S") - dt_util.now().replace(
-        tzinfo=None
-    )
-
-    return str(int(diff.total_seconds() / 60))
-
-
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the Dublin public transport sensor."""
-    name = config[CONF_NAME]
-    stop = config[CONF_STOP_ID]
-    route = config[CONF_ROUTE]
+    name = config.get(CONF_NAME)
+    stop = config.get(CONF_STOP_ID)
+    route = config.get(CONF_ROUTE)
 
     data = PublicTransportData(stop, route)
     add_entities([DublinPublicTransportSensor(data, stop, route, name)], True)
```

Comment:
`stop` and `route` is not used in `DublinPublicTransportSensor` and can be removed.